packages/superset-ui-legacy-plugin-chart-heatmap/src/Heatmap.css

```diff
@@ -43,13 +43,14 @@
   shape-rendering: crispEdges;
 }
 
-.superset-legacy-chart-heatmap canvas, .superset-legacy-chart-heatmap img {
-  image-rendering: optimizeSpeed;             /* Older versions of FF          */
-  image-rendering: -moz-crisp-edges;          /* FF 6.0+                       */
+.superset-legacy-chart-heatmap canvas,
+.superset-legacy-chart-heatmap img {
+  image-rendering: optimizeSpeed; /* Older versions of FF          */
+  image-rendering: -moz-crisp-edges; /* FF 6.0+                       */
   image-rendering: -webkit-optimize-contrast; /* Safari                        */
-  image-rendering: -o-crisp-edges;            /* OS X & Windows Opera (12.02+) */
-  image-rendering: pixelated;                 /* Awesome future-browsers       */
-  -ms-interpolation-mode: nearest-neighbor;   /* IE                            */
+  image-rendering: -o-crisp-edges; /* OS X & Windows Opera (12.02+) */
+  image-rendering: pixelated; /* Awesome future-browsers       */
+  -ms-interpolation-mode: nearest-neighbor; /* IE                            */
```

Comment:
Surprised Prettier isn't smart enough to convert these to `//` comments!

Reply:
I don't think that works in `css`

Reply:
Hah, you're right... everything is (LE/SA)SS in my head by default :P